src/modules/testimonials/testimonial_service.ts

```diff
@@ -17,4 +17,22 @@ export namespace Testimonial {
 
     return result
   }
+
+  export const findAllUsingCursor = async (requestQuery: Entity.RequestQueryUsingCursor): Promise<Entity.ResponseFindAllUsingCursor> => {
+    const items: any = await Repository.findAllUsingCursor({
+      type: requestQuery.type,
+      dateBefore: requestQuery?.next_page ? new Date(requestQuery.next_page) : new Date(),
+      perPage: Number(requestQuery?.per_page) || 3,
+    })
+
+    const result: Entity.ResponseFindAllUsingCursor = {
+      data: items,
+      meta: {
+        next_page: items.length ? items[items.length - 1].created_at : null,
```

Comment:
it's better to put it into the itemsLength variable, so it doesn't fit in calculating length

Reply:
ok then,